wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -154,6 +154,7 @@ public WikiServiceImpl(ConfigurationManager configManager,
 
   @Override
   public void start() {
+    initTemplates();

```

Comment:
That does not seem logic. Wiki creations should be done with the method createWiki(). Is it due to this commit : https://github.com/exodev/wiki/commit/8b0552651ab4eb029f54837f86b161a8a9a45f98 ? It creates a wiki in a read method (getWikiObject), and directly by calling the data storage layer. Creation of the wiki should be done at service level.